lightning/src/ln/peer_handler.rs

```diff
@@ -173,6 +194,18 @@ pub struct MessageHandler<CM: Deref, RM: Deref> where
 	pub route_handler: RM,
 }
 
+/// Reader for messages external to the LN protocol.
```

Comment:
nit: I would say "unrelated to the channel/gossip layers" as those custom/unknown messages are still defined in the terms of BOLT 1 and 8.

Reply:
Used your phrasing in new version